astropy/coordinates/tests/test_angles.py

```diff
@@ -572,6 +572,12 @@ def test_angle_string():
     a = Angle("10'")
     assert_allclose(a.value, 10.0)
     assert a.unit == u.arcminute
+    a = Angle('00:00:60N', u.deg)
```

Comment:
Could you also add tests for E, W, for angles in degrees (i.e., no colons), etc.? Also very important to add tests that wrong input does not work, i.e., that `ValueError` is raised for, say `-10S`, or `+10N` or `11:12:13NE`. What also should fail is NS on right ascensions (e.g., hms angles like `10h20mN`) while EW on dms angles is fine (e.g., `-10dN`)

Reply:
+10N will be working because according to the defined grammar +10N is same as 10N there is no way to find out whether that positive 1 is coming due to no sign or due to positive ahead of the angle.